src/index.html

```diff
@@ -5,8 +5,395 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#"
+        class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple"
+              class="nav__link is-active"
+            >
+            Apple
+            </a>
```

Comment:
Need to fix code style everywhere
```suggestion
            <a href="#apple"class="nav__link is-active">
              Apple
            </a>
```